tools/accuracy_checker/accuracy_checker/adapters/pose_estimation.py

```diff
@@ -18,13 +18,17 @@
 from operator import itemgetter
 
 import cv2
+import ngraph as ng
```

Comment:
please remove all runtime dependencies from adapter (ngraph, openvino, torch).
this API should be clear and independent from runtime. If you need some additional logic conected witr inference, that probably you need write custom evaluator

Reply:
This is a bit weird constraint, but those are not needed anymore. Removed.